docs/user/security/securing-kibana.asciidoc

```diff
@@ -5,79 +5,62 @@
 <titleabbrev>Configure security</titleabbrev>
 ++++
 
-{kib} users have to log in when {stack-security-features} are enabled on your
-cluster. You configure roles for your {kib} users to control what data those
-users can access.
+When you start {es} for the first time, {stack-security-features} are enabled on
+your cluster and TLS is configured automatically. The security configuration
+process generates a password for the `elastic` user and an enrollment token for
+{kib}.
+{ref}/configuring-stack-security.html[Start the {stack} with security enabled]
+and then enroll {kib} as part of the configuration process.
 
-Most requests made through {kib} to {es} are authenticated by using the
-credentials of the logged-in user. There are, however, a few internal requests
-that the {kib} server needs to make to the {es} cluster. For this reason, you
-must configure credentials for the {kib} server to use for those requests.
+You can then log in to {kib} as the `elastic` user to create additional roles
+and users. You can also configure <<elasticsearch-mutual-tls,mutual TLS>>
+between {kib} and {es} where requests are authenticated by both applications.
```

Comment:
I'd suggest removing the sentence about mutual TLS, it puts too much emphasis on this feature that most folks won't need, and we already link to that page in step (3) below.

Reply:
++ I'll remove that sentence.